contrib/win32/win32compat/inc/dlfcn.h

```diff
@@ -8,4 +8,4 @@ HMODULE dlopen(const char *filename, int flags);
 
 int dlclose(HMODULE handle);
 
-FARPROC dlsym(HMODULE handle, const char *symbol);
\ No newline at end of file
+void * __stdcall dlsym(HMODULE handle, const char *symbol);
```

Comment:
Does this need to be stdcall? It would probably make more sense to leave this with the default calling convention like the other functions in this file.

Reply:
You're right.  We do not.  Commit adjusted.